README.md

```diff
@@ -1,304 +1,125 @@
+* Click config which opens a new window for settings.
+* Enter the credentials for your cloud:
+  - Go to [S3 Storage Configuration](https://github.com/Seagate/cloudfuse/wiki/S3-Storage-Configuration) to find S3 credential requirements.
+  - Go to [Azure Storage Configuration](https://github.com/Seagate/cloudfuse/wiki/Azure-Storage-Configuration) to find Azure credential requirements.
+* Select you desired pipeline mode:
+  - File Caching: Browse to a folder you wish to have you temp file caching.
+  - Streaming: If 0MB are selected, streaming will not cache any blocks.
+* Save your changes for the GUI to write your new config file.
 
-## Features compared to blobfuse2
-- Supports any S3 compatible storage
-- Adds a GUI to configure and start mounts
-- Runs on Windows using WinFSP in foreground or as a Windows service
-
-## Download Cloudfuse
-You can install Cloudfuse by cloning this repository. In the workspace execute
-the build script `./build.sh` to build the binary. This will build a binary both
-for Linux or for Windows depending on the OS you are using.
+If you wish to manually configure the config file without the GUI, you can find instructions [here](https://github.com/Seagate/cloudfuse/wiki/Config-File).
 
+## Basic Use
 ### Linux
-Cloudfuse currently only supports libfuse2. On Linux, you need to install the
-libfuse2 package, for example on Ubuntu:
-
-    sudo apt install libfuse2
-
-#### Running on Linux
-To start your mount of an S3 Bucket or Azure Container use the `mount` command
-and specify the location of your config file. See [config file](#config-file)
-for information about the config file. By default, cloudfuse will run in the
-background which allows you to close the terminal when you start a mount. If you
-would like it to run in the foreground you can specify `foreground: true` in
-your config file or pass `--foreground=true` as an argument when mounting.
-
-        cloudfuse mount <mount path> --config-file=<config file>
+Launch the provided GUI and check the configs are set from the above instructions in Config.
+* `To mount`
+  - Through the main window in the GUI, browse to the location you want your cloud to be mounted, then select the EMPTY folder you want.
+  - In the drop down menu, select the type of cloud you have, either an S3 Bucket or Azure Blob Storage, then mount the cloud.
+  - You should now see your data in the folder you selected.
+* `To unmount`
+  - Through the main window in the GUI, browse to the folder your mounted cloud resides and select it.
+  - Select the type of cloud you have, S3 or Azure.
+  - Click the unmount mutton.
+  - The folder the cloud was in should now be empty.
 
 ### Windows
-On Windows, you also need to install the third party utility
-[WinFsp](https://winfsp.dev/). To download WinFsp, please run the WinFsp
-installer found [here](https://winfsp.dev/rel/).
+Note: Make sure WinSFP is installed in the setup section dedicated to Windows.

```

Comment:
Is this referring to the `Installation` section? I know it doesn't have subsections yet, but can you link that here? Something like this:
`in the [setup section dedicated to Windows](/Installation).`

Reply:
Yes, that was what I was referring to.